core/field.js

```diff
@@ -899,6 +899,7 @@ Blockly.Field.prototype.setValue = function(newValue) {
   var oldValue = this.getValue();
   if (oldValue === newValue) {
     doLogging && console.log('same, return');
```

Comment:
I think this needs to be updated to reflect the change. Something like 'same, doValueUpdate_ then return'.

Reply:
Fixed it! :smile: thank you for the input!